_posts/2021-09-21-release-1.14.0.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post 
+title:  "Apache Flink 1.14.0 Release Announcement"
+date: 2021-09-21T08:00:00.000Z 
+categories: news 
+authors:
+- joemoe:
+  name: "Johannes Moser"
+
+excerpt: The Apache Flink community is excited to announce the release of Flink 1.14.0! Around xxx contributors worked on over xxxx issues to TODO.
+---
+
+Just a couple of days ago the Apache Software Foundation announced it’s annual report and Apache
+Flink being in the Top 5 in all relevant categories is just an outcome of the work of the
+community (that has been done yet again) for 1.14.0. The consistency how this project is moving
+forward is remarkable. Once again 200 plus contributors worked on over 1,000 issues.
+
+Apache Flink not only supports batch and stream processing, but has been always following the goal
+of making it a unified experience. With Apache Flink 1.14.0 batch and stream processing moved closer
+together. The first sinks and sources are now providing a unified API (following FLIP-27 and
+FLIP-143). Hybrid source has been introduced. The DataStream batch mode has been pushed to the
+TableAPI. Under the hood checkpoints are allowed even after tasks are finished truly enabling mixed
+or bounded jobs. Existing features have been haromised throughout all available APIs. From
+DataStream to TableAPI and SQL and vice versa. The DataStream batch mode is maturing after its
```

Comment:
```suggestion
Table API. Under the hood checkpoints are allowed even after tasks are finished truly enabling mixed
or bounded jobs. Existing features have been haromised throughout all available APIs. From
DataStream to Table API and SQL and vice versa. The DataStream batch mode is maturing after its
```
All the other occurrences of Table API are written with a space in this blog post.